src/docs/style/themes.mdx

```diff
@@ -22,42 +21,80 @@ One of the key benefits of PX Blue is consistency of look & feel between differe
         style={{ marginRight: 0 }}
     />
 </div>
+
+## Explore Our Themes
+
+Have a look at the component theme explorer below and learn how PX Blue themes work on a high level. The theme explorer selectively showcases some of the most representative components from Material Design and from our component library that fabrics our theme.
+
+<ThemeBrowser />
+
+You are also encouraged to visit our code examples, modify our code, and see the magic happens in real time.
+
+<React.Fragment>
+    <DemoCard
+        repository={'showcase-demo'}
+        angular={`https://stackblitz.com/github/pxblue/angular-showcase-demo/tree/master`}
+        react={`https://codesandbox.io/s/github/pxblue/react-showcase-demo/tree/master`}
+    />
+    <Divider />
+</React.Fragment>
+
+# Blue Theme / Light Theme
+
+> A SCREENSHOT HERE
+
+The blue theme, inherited and expanded from Material Design's baseline theme, is the default theme for all PX Blue applications. With a clean color vibrance and high text legibility, it is designed to suit day to day use on a computer monitor, on-site facility inspection with a tablet, and on a mobile phone screen. It should be applied at the root level of your app.
```

Comment:
Technically all of our themes are based on the MD baseline theme...I don't think we need to explicitly say that.

Reply:
I've moved it to the top. 

The term "theme" technically consists of colors, typographies, shapes, icons, etc. I find it worth mentioning that our theme is no different from their baseline theme except for colors and typographies (design patterns are not part of a theme), as part of an answer to the question "what's the diff between MD and PXB"